### DISCUSSION TITLE
[Q/A][Java SDK] how to disable register  in junit test

### DISCUSSION THREAD

**Discussion by yjant** [apache/dubbo#14290] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

Hello, I use dubbo 3.0.6 and nacos  as  registration center  in my springboot. When the application start, dubbo will register to nacos. Is it necessary to do register  in junit test? this process take a long time, how can I disable  register  in junit test?  
I tried like this, but it doesnt make sense

     @TestPropertySource(properties = {
      
        "dubbo.register.enabled=false",
        "dubbo.registry.address="
    }

In shenyu, I can use `shenyu.register.enabled=false` to disable register, Is there something like that  in dubbo?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
`dubbo.regitry.register=false`
